Assets/Scripts/Battlefield/StateBehaviors/StrikerTurnStateBehavior.cs

```diff
@@ -26,19 +31,21 @@ override public void OnStateEnter(Animator animator, AnimatorStateInfo stateInfo
                 playerToAttack = i;
             }
         }
-        Tile target = brain.manager.playerUnits[playerToAttack].GetComponent<UniqueCreature>().currentTile;
+        Tile targetedPlayer = brain.manager.playerUnits[playerToAttack].GetComponent<UniqueCreature>().currentTile;
+        target = grid.tiles[targetedPlayer.x + 1, targetedPlayer.y];
     }
 
     // OnStateUpdate is called on each Update frame between OnStateEnter and OnStateExit callbacks
     override public void OnStateUpdate(Animator animator, AnimatorStateInfo stateInfo, int layerIndex)
     {
         brain.outline.OutlineColor = Color.yellow;
-        if (target)
+        if (target && target.unitOnTile == null)
         {
-            brain.creature.Move(target);
+            List<Tile> path = star.FindPath(target, brain.creature.gridHolder, brain.creature);
+            brain.creature.FollowPath(path);
         }
 
-        if (target && Vector3.Distance(animator.transform.position, target.GetCenterOfTile()) < 2f)
+        if (target && Vector3.Distance(animator.transform.position, target.GetCenterOfTile()) < 1f)
         {
             brain.isMyTurn = false;
         }
```

Comment:
Striker sounds hype, but i see that this code is written in many places such as findPath and Moving and is simply better is it was written once and the Strategy was given to both the player and Enemy. This allows for us to not have to duplicate code and use better abstraction for when this update loop get much much larger. Something like the MovementSystem again would be perfect here instead of relying on a concrete single implementation move system using inheritance from the CreatureBase. Composition and dependency inversion serves our purposes much better here.